src/main/java/org/umssdiplo/automationv01/core/managepage/audit/AuditList.java

```diff
@@ -0,0 +1,24 @@
+package org.umssdiplo.automationv01.core.managepage.audit;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.umssdiplo.automationv01.core.managepage.BasePage;
+import org.umssdiplo.automationv01.core.utils.CommonEvents;
+
+/**
+ * @author Neyber Rojas Zapata
+ */
+public class AuditList extends BasePage {
```

Comment:
delete please

Reply:
This class was created for validate the new audit creation button. If the button exists, then the audit list is shown.